app/components/image-zoom.js

```diff
@@ -22,10 +23,10 @@ export default Ember.Component.extend({
 
   didInsertElement() {
     if (!this.get("isMobileApp")) {
-      var _this = this;
+      const parentId = `#${this.get("parentId")}`;
```

Comment:
We don't want to have any references to IDs in components. It makes them much harder to reuse.

I would recommend you use the jquery `.parent()` method to access the parent.

e.g

```
this.$().parent().lightGallery({
   ...
});
```

I don't really get why we need to access the parent to be honest, we could likely just create the gallery in the component's div.

Reply:
Ahh.. it makes sense to use component default `div` (since components default tag is div). Will try it out and update and test it out. Thanks for pointing it out.